theme/forms.py

```diff
@@ -236,6 +236,7 @@ def __init__(self, request, *args, **kwargs):
         super(SignupForm, self).__init__(*args, **kwargs)
 
     def verify_captcha(self):
+        return (True, [])
```

Comment:
Remove this. It is in your debug local environment to disable captcha verification. Although we don't use recaptcha any more, try not to get debug code in our codebase.